recipes-kernel/linux/linux-smelt/defconfig

```diff
@@ -3361,7 +3294,7 @@ CONFIG_ENCRYPTED_KEYS=y
 # CONFIG_SECURITY_DMESG_RESTRICT is not set
 # CONFIG_SECURITY is not set
 # CONFIG_SECURITYFS is not set
-# CONFIG_DEFAULT_SECURITY_DAC is not set
+CONFIG_DEFAULT_SECURITY_DAC=y
```

Comment:
Why do you need this?

Reply:
Eh, not sure. Must have been automatically enabled itself. while messing with `menuconfig`.
My process on defconfig changes is:
```sh
bitbake -c menuconfig linux-smelt
# Adjust the kernel config
# Let it compile, test if it runs. look for obvious issues in dmesg.
# Copy from `work-shared` .config to defconfig
```

Will look into this.

Reply:
This is actually set by the Kconfig of security:
```
config DEFAULT_SECURITY
        string
        default "selinux" if DEFAULT_SECURITY_SELINUX
        default "smack" if DEFAULT_SECURITY_SMACK
        default "tomoyo" if DEFAULT_SECURITY_TOMOYO
        default "apparmor" if DEFAULT_SECURITY_APPARMOR
        default "yama" if DEFAULT_SECURITY_YAMA
        default "" if DEFAULT_SECURITY_DAC

endmenu
```

And we have `CONFIG_DEFAULT_SECURITY=""`

Reply:
Mhh okay fair enough. We don't have CONFIG_SECURITY set anyway so there shouldn't be any LSM. That's fine